config/flake8.ini

```diff
@@ -0,0 +1,50 @@
+[flake8]
+exclude = fixtures,docs,site
+max-line-length = 132
+strictness = long
+ignore =
+    # we write docstrings in markdown, not rst
+    RST*,
+    # redundant with W0622 (builtin override), which is more precise about line number
+    A001,
+    # missing docstring in magic method
+    D105,
+    # multi-line docstring summary should start at the first line
+    D212,
+    # whitespace before ‘:’ (incompatible with Black)
+    E203,
+    # redundant with E0602 (undefined variable)
+    F821,
+    # black already deals with quoting
+    Q000,
+    # use of assert
+    S101,
+    # we are not parsing XML
+    S405,
+    # line break before binary operator (incompatible with Black)
+    W503,
+    "-C0103", # two-lowercase-letters variable DO conform to snake_case naming style
```

Comment:
Any reason you're adding the comment at the end of this line and not the others?
If possible I'd to have comments on the same lines, even if codes must be wrapped in double-quotes. What do you think?

Reply:
Also why does it have the `-` prefix and not the others?

Reply:
I... have no idea how this appeared here.

And no, inline comments are not possible.
https://flake8.pycqa.org/en/3.8.4/user/configuration.html#project-configuration